google/cloud/internal/backoff_policy.cc

```diff
@@ -24,33 +24,32 @@ std::unique_ptr<BackoffPolicy> ExponentialBackoffPolicy::clone() const {
   auto tmp =
       google::cloud::internal::make_unique<ExponentialBackoffPolicy>(*this);
   // Force OnCompletion() to reseed the generator.
-  tmp->generator_seeded_ = false;
+  tmp->generator_.reset();
```

Comment:
Shouldn't we override the copy ctor to do this too?

Reply:
Then we would not need to do it here, of course.  But then we would also need to change the copy-assignment, and then we would need to document why, because both would be "weird".  And then the only place where this matters (i.e., where using a copy of the PRNG is not right) is in this `clone()` function.

If you folks feel strongly I can change it, but sounds like more work, for the same value.